Ontology/Space/Zone/OccupancyZone.json

```diff
@@ -0,0 +1,9 @@
+{
+  "@id": "dtmi:digitaltwins:rec_3_3:core:OccupancyZone;1",
+  "@type": "Interface",
+  "displayName": {
+    "en": "Occupancy Zone"
+  },
+  "extends": "dtmi:digitaltwins:rec_3_3:core:Zone;1",
+  "@context": "dtmi:dtdl:context;2"
```

Comment:
I would add a description to this class to eliminate confusion.

Reply:
Let me add that in the next commit